ios/Classes/TiAirshipModule.m

```diff
@@ -118,6 +118,26 @@ - (void)setUserNotificationsEnabled:(id)value {
     [[UAirship push] updateRegistration];
 }
 
+- (id)isAutoBadgeEnabled {
+    return NUMBOOL([UAirship push].autobadgeEnabled);
+}
+
+- (void)setIsAutoBadgeEnabled:(id)args {
+    [UAirship push].autobadgeEnabled = [TiUtils boolValue:args def:YES];
+}
+
+- (id)badgeNumber {
+    return NUMINTEGER([UAirship push].badgeNumber);
+}
+
+- (void)setBadgeNumber:(id)args {
+    [UAirship push].badgeNumber = [TiUtils boolValue:args def:YES];
```

Comment:
Shouldn't this be an int rather than a bool?

Reply:
yeah, i was testing with 1 so i didnt notice...